Core/Core/Api/GraphQL/Client.cs

```diff
@@ -160,58 +136,28 @@ public async Task<T> ExecuteGraphQLRequest<T>(GraphQLRequest request, Cancellati
           return result.Data;
         })
         .ConfigureAwait(false);
-      success = true;
-      return result;
-    }
-    // cancellations are bubbling up with no logging
-    catch (OperationCanceledException)
-    {
-      throw;
     }
-    // we catch forbidden to rethrow, making sure its not logged.
-    catch (SpeckleGraphQLForbiddenException<T>)
+    catch (Exception ex)
     {
+      exception = ex;
       throw;
     }
-    // anything else related to graphql gets logged
-    catch (SpeckleGraphQLException<T> gqlException)
-    {
-      SpeckleLog.Logger
-        .ForContext("graphqlResponse", gqlException.Response)
-        .ForContext("graphqlExtensions", gqlException.Extensions)
-        .ForContext("graphqlErrorMessages", gqlException.ErrorMessages.ToList())
-        .Warning(
-          gqlException,
-          "Execution of the graphql request to get {resultType} failed with {graphqlExceptionType} {exceptionMessage}.",
-          typeof(T).Name,
-          gqlException.GetType().Name,
-          gqlException.Message
-        );
-      throw;
-    }
-    // we log and wrap anything that is not a graphql exception.
-    // this makes sure, that any graphql operation only throws SpeckleGraphQLExceptions
-    catch (Exception ex) when (!ex.IsFatal())
-    {
-      SpeckleLog.Logger.Warning(
-        ex,
-        "Execution of the graphql request to get {resultType} failed without a graphql response. Cause {exceptionMessage}",
-        typeof(T).Name,
-        ex.Message
-      );
-      throw new SpeckleGraphQLException<T>("The graphql request failed without a graphql response", ex, request, null);
-    }
     finally
     {
-      // this is a performance metric log operation
-      // this makes sure that both success and failed operations report
-      // the same performance log
-      timer.Stop();
-      var status = success ? "succeeded" : "failed";
-      SpeckleLog.Logger.Information(
-        "Execution of graphql request to get {resultType} {resultStatus} after {elapsed} seconds",
+      LogEventLevel logLevel = exception switch
```

Comment:
Hopefully this section is clear what it does. If not I'll explain over call. 
The goal was to have more sensible log levels, have fewer logs per request, and to avoid unintentional wrapping of unexpected exception types